src/video/cocoa/SDL_cocoaevents.m

```diff
@@ -420,6 +427,10 @@ static void CreateApplicationMenus(void)
 
     /* Add menu items */
     title = [@"About " stringByAppendingString:appName];
+
+    // !!! FIXME: Menu items can't take parameters, just a basic selector, so this should instead call a selector
+    // !!! FIXME: that itself calls -[NSApplication orderFrontStandardAboutPanelWithOptions:optionsDictionary],
+    // !!! FIXME: filling in that NSDictionary with pieces of SDL_GetAppMetadata()'s properties.
```

Comment:
This happens before we call SDL_main(), right?

Reply:
It happens in video init.